### ISSUE TITLE
Unable to "npm install -g thor"

### ISSUE THREAD

**Issue by john-doherty** [observing/thor#45]
I get the following error on a mac:

Command failed: git clone --template=/Users/JD/.npm/_git-remotes/_templates --mirror git://github.com/3rd-Eden/ws.git /Users/JD/.npm/_git-remotes/git-github-com-3rd-Eden-ws-git-2e9f11e5

**Comment by ankitiitb1069** [comment]
Even I am having the same error on my ubuntu 16.04

**Comment by mcary** [comment]
Seems this dependency has been removed?  Is there another variation available?

```
npm ERR! git clone git://github.com/3rd-Eden/ws.git fatal: remote error: 
npm ERR! git clone git://github.com/3rd-Eden/ws.git   Repository not found.
npm ERR! Error: Command failed: Cloning into bare repository '/Users/marcel/.npm/_git-remotes/git-github-com-3rd-Eden-ws-git-3a6cde0f'...
npm ERR! fatal: remote error: 
npm ERR!   Repository not found.
```

**Comment by john-doherty** [comment]
Anyone found a fix for this?

**Comment by rszalski** [comment]
@john-doherty yes

TL; DR;

Install the package from the newest source:

1. `git clone git@github.com:observing/thor.git`
2. `cd thor/`
3. `npm instlal -g .`

It seems that in the `HEAD` version of `thor` has a fixed dependency on the [ws](https://github.com/websockets/ws) package. The commit that changed this was https://github.com/observing/thor/commit/6940acfb22b8d6334139f199c22e0e4b25a84297. It also seems that this was not a fix for a missing repo, just an ordinary switch to a different fork (that itself has more fixes).

**Comment by john-doherty** [comment]
Unable to clone, getting the following error:

```
Cloning into 'thor'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by rszalski** [comment]
@john-doherty Look's like you don't have a SSH key uploaded to Github or added to ssh-agent (https://help.github.com/articles/connecting-to-github-with-ssh/).

Try with HTTPS: `git clone https://github.com/observing/thor.git`

**Comment by alecz20** [comment]
Seeing as Node.js is the only mentioned dependency, I installed the stable Node.js on Windows, and then ran the command `npm install -g thor` but it failed saying git is not found.

```
...
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! code ENOGIT

npm ERR! not found: git
npm ERR!
npm ERR! Failed using git.
...
```

Since git is not mentioned as a dependency, I figured I didn't install Node.js correctly, but installing other Websocket packages works. Should Open a separate issue?

**Comment by rszalski** [comment]
@alecz20 Indeed Git is required. You could modify the README to mention this explicitly (via PR, will be faster than opening an issue :). To fix this, you just need Git installed on your Windows box, see: https://github.com/npm/npm/wiki/Troubleshooting#no-git

**Comment by krcgk** [comment]
👍

**Comment by dmirogin** [comment]
`npm install -g https://github.com/observing/thor`

**Comment by ihemantkumar** [comment]
I somehow installed the thor [this is how:](https://github.com/observing/thor/issues/55#issuecomment-650181511) but it is not working for me.
`# thor --amount 2000 ws://localhost:8443 wss://localhost:8443
/usr/bin/env: node: No such file or directory
`
I am having this issue. Does anyone know how to fix this?
